src/components/ButtonNav/styled.ts

```diff
@@ -18,11 +19,7 @@ export const Wrapper = styled.nav`
 	}
 `
 
-type Props = {
-	color: string
-}
-
-export const StyledButton = styled(Button)<Props>`
+export const StyledButton = styled(Button)<{ $theme: Theme }>`
```

Comment:
What's your opinion for this naming?
```suggestion
export const StyledButton = styled(Button)<{ selectedTheme: Theme }>`
```

Reply:
This naming with `$` at the beginning has specific purpose. :sweat_smile:
https://styled-components.com/docs/api#transient-props

Reply:
Ok :)